spec/twilio_messager_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'twilio_messager'
+
+describe Messager do
+  
+  subject(:messager)   { described_class.new }
+  
+  it 'sends a confirmation sms to the client' do
+    allow(messager).to receive(:send_sms).and_return('Thank You! Your order will be delivered')
```

Comment:
Metrics/LineLength: Line is too long. [95/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)